crates/core/src/db/datastore/locking_tx_datastore/mod.rs

```diff
@@ -1078,7 +1078,9 @@ impl MutTxId {
         // we have created the table in the database, but have not yet
         // represented in memory or inserted any rows into it.
         Ok(Cow::Owned(
-            self.schema_for_table(table_id, database_address)?.get_row_type(),
+            self.schema_for_table(table_id, database_address)?
+                .get_row_type()
+                .clone(),
```

Comment:
Rust complains:

```
error[E0507]: cannot move out of dereference of `spacetimedb_sats::db::def::TableSchema`
   --> crates/core/src/db/datastore/locking_tx_datastore/mod.rs:274:61
    |
274 |         for col in system_tables().into_iter().flat_map(|x| x.columns) {
    |                                                             ^^^^^^^^^ move occurs because value has type `std::vec::Vec<spacetimedb_sats::db::def::ColumnSchema>`, which does not implement the `Copy` trait
```

I think this is a consequence of the `read_only` mark.